src/zoid/buttons/util.js

```diff
@@ -18,6 +18,38 @@ export function determineFlow(props : ButtonProps) : $Values<typeof BUTTON_FLOW>
     }
 }
 
+export function supportsQRPay(funding : $Values<typeof FUNDING>) : boolean {
+    if (funding === FUNDING.VENMO && !isIos() && !isAndroid()) {
+        return true;
+    }
+
+    return false;
+}
+
+/* eslint-disable flowtype/require-exact-type */
```

Comment:
Nice work with using `ownerDocument`. I wonder if there's a built-in Flow DOM type that would play nice with this approach. What error did you run into when trying the `HTMLElement` type?

Reply:
Since the `EventTarget` isn't explicitly an `HTMLElement`[ in general built-in DOM typing](https://stackoverflow.com/questions/28900077/why-is-event-target-not-element-in-typescript), it's always going to throw an error when I try to call `ownerDocument` regardless if I try to pass in `event.target` typed as `HTMLElement` or event as `Event` and derive it as `event.target.ownerDocument`. Since the context of event is needed to drill into the frame, I figured it made the most sense to just pass the event rather then the target.

Reply:
Thanks for explaining. I'm okay with this workaround to please flow.